src/lib/context/downloads.ts

```diff
@@ -38,21 +41,41 @@ const isVersionStable = async (
   return false;
 };
 
+const isVersionExperimentalStable = async (
+  project: string,
+  version: string
+): Promise<boolean> => {
+  const { builds } = await getVersionBuilds(project, version);
+  for (let i = builds.length - 1; i >= 0; i--) {
+    if (builds[i].channel === "experimental") return true;
+  }
+
+  return false;
+};
+
 export const getProjectProps = (id: string): GetStaticProps => {
   return async () => {
     const { project_name, versions, version_groups } = await getProject(id);
 
-    let latestVersion = versions[versions.length - 1];
+    let latestStableVersion = versions[versions.length - 1];
+    let latestExperimentalVersion = null;
     for (let i = versions.length - 1; i >= 0; i--) {
       if (await isVersionStable(id, versions[i])) {
-        latestVersion = versions[i];
+        latestStableVersion = versions[i];
         break;
       }
+      if (
+        !latestExperimentalVersion &&
+        (await isVersionExperimentalStable(id, versions[i]))
+      ) {
+        latestExperimentalVersion = versions[i];
+      }
     }
 
     const project = {
```

Comment:
Can we not assign `latestExperimentalVersion` when `latestStableVersion != versions[versions.length - 1]`? That way we can avoid making more network requests